libs/server/Resp/Parser/RespCommand.cs

```diff
@@ -97,6 +97,7 @@ public enum RespCommand : byte
         HSETNX,
         INCR,
         INCRBY,
+        INCRBYFLOAT,
```

Comment:
Please add a BDN for `IncrementByFloat` in `benchmark\BDN.benchmark\Resp\RespParseStress.cs` - following the same pattern as `Increment()` in that `RespParseStress` class.

Reply:
RespIncrementStress is less relevant, we are not really using that at the moment.

Reply:
In RespParseStress:
* Create a separate command type for IncrementByFloat
* Add the setup logic to create the original value of key such as f => 1.1
* The stress will issue a command such as `INCRBYFLOAT f 1.1` or similar

Reply:
Actually, do not worry about adding the BDN for this. There are some subtle issues with it that will be resolved by an open PR and then we can add it.